Formula/arm-cli.rb

```diff
@@ -0,0 +1,19 @@
+class ArmCli < Formula
+  desc "Armory ARM CLI"
+  homepage "https://github.com/armory-io/arm/"
+  url "https://github.com/armory-io/arm/releases/download/2.3.0/arm-2.3.0-macos-arm64.zip"
+  sha256 "c34c1d9a8ca92b2a449412680198d23d4b50ca08fd97ae7a0afaa49113a129b9"
+  license "Apache-2.0"
+
+  def install
+    bin.install "arm-2.3.0-darwin-arm64" => "arm"
+  end
+
+  test do
+    run_output = shell_output("#{bin}/arm 2>&1")
+    assert_match "The Armory Platform CLI", run_output
+
+    version_output = shell_output("#{bin}/arm version 2>&1")
+    assert_match "rainmaker", version_output
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Yep. The remaining issue seems to be with the source being a binary package, which I cannot change at the moment.

Reply:
Then it's not eligible for inclusion in Homebrew.